Pymoe/Mal/Abstractions.py

```diff
@@ -8,8 +8,8 @@ def __init__(self, current, total):
         self.current = current
         self.total = total
 
-    def __str__(self):
-        return 'NT_EPISODES(current={},total={})'.format(self.current, self.total)
+    def __repr__(self):
+        return 'NT_EPISODES(current={}, total={})'.format(repr(self.current), repr(self.total))
```

Comment:
Just want to point out the reason that there are calls like `repr(self.current)` is because `__repr__` is supposed to return a string such that `eval(repr(x))` returns an object equivalent to `x`. To ensure that, the values of the fields (`current` and `total` here) in the string need to be valid `__repr__` strings, too.